docs/advanced/full-node.md

```diff
@@ -0,0 +1,63 @@
+# <img class="dcr-icon" src="/img/dcr-icons/AtoB.svg" /> Running a Full Node
+
+---
+
+This guide explains everything you need to know about running a full node on Decred.
+
+## What is a full node?
+
+A full node is a program that fully validates transactions and blocks, without having to rely on a third party. For this guide, we'll be using [dcrd](https://github.com/decred/dcrd), the most popular full node implementation for Decred. 
+
+## Why run a full node?
+
+There are a number of reasons to run a full node:
+
+- **Help the network:** Full nodes are the backbone of Decred. By validating and broadcasting transactions without relying on third parties&mdash; as lightweight nodes do&mdash;they help keep the network decentralized. Each full node added to the network adds to its security and robustness. 
+- **Safety & Privacy:** Because a lightweight node cannot fully validate transactions and blocks, it must trust third parties to do so. This means that the third party will have access to information about your transactions, as well as any personal data you give to that third party. For instance, many lightweight wallets rely on a centralized server to process transactions. Even Simplified Payment Verification (SPV) wallets, which typically connect directly to full nodes in the network, still rely on those other full nodes to process and broadcast their transactions. 
+
+## Requirements:
+
+Below are the basic requirements for running a full node:
+
+- \>15 GB disk space (as of September 2019, increases over time)
+- \>2 GB of RAM
+- Stable Operating System (OS) &mdash;Windows 7/8.x/10 (server preferred), macOS, Linux
```

Comment:
Wasn't sure about this line actually. Do we need it at all? Mac, Windows, Linux kind of covers nearly all OSes really, so why bother listing them?

Reply:
Agree, its quite pointless.

We talk about OS choices in this page https://docs.decred.org/advanced/general-security/
maybe a link there? I dont mind either way

Reply:
Right now have just stripped out the 'stable' part:

>- Operating System (OS) &mdash;Windows 7/8.x/10 (server preferred), macOS, Linux

But maybe we can just say 'Supported Operating System (OS)?'. Will dcrinstall work on any OS? This line might not be very relevant in this context, but I wonder if readers will want to see something about OS requirements anyway, just because they're use to seeing it?

Reply:
Yes dcrinstall works on mac/win/linux. I dont really mind tbh, either listing them or just writing "supported operating systems" is fine

Reply:
Have this now:

>- Supported Operating System (OS) &mdash;Windows 7/8.x/10 (server preferred), macOS, Linux